consortia/environments/values-dev.yaml

```diff
@@ -235,7 +239,11 @@ backend:
       clientId: "<path:portal/data/processes-worker#offerprovider-client-id>"
       clientSecret: "<path:portal/data/dev/processes-worker#offerprovider-client-secret>"
     onboardingServiceProvider:
-      encryptionKey: "<path:portal/data/dev/processes-worker#process-onboardingserviceprovider-encryption-key>"
+      encryptionConfigs:
```

Comment:
Please provide a description about what to consider when upgrading from the previous encryption key, here as a comment and in the version upgrade file in portal-assets repository.